velox/docs/functions/json.rst

```diff
@@ -2,6 +2,95 @@
 JSON Functions
 ==============
 
+JSON Format
+---------------
+JSON is a language-independent data format that represents data as
+human-readable text. A JSON text can represent a number, a boolean, a
+string, an array, an object, or a null, with slightly different grammar.
+For instance, a JSON text representing a string must escape all characters
+and enclose the string in double quotes, such as ``"123\n"``, whereas a JSON
+text representing a number does not need to, such as ``123``. A JSON text
+representing an array must enclose the array elements in square brackets,
+such as ``[1,2,3]``. More detailed grammar can be found on https://www.json.org.
+
+JSON Vectors
+---------------
+There are a number of Presto JSON functions expecting JSON-type inputs. An
```

Comment:
Not only there are functions that take JSON as input, but there are also operators (cast) that return JSON as output. I wonder if there is a way to clarify that. Maybe, we could avoid talking about BaseVector::create here (non-user friendly terms), but rather say that vectors of JSON type are implemented as vector of strings.

Reply:
I talk about functions that expect JSON inputs or return JSON outputs. I also mentioned that reading and writing to JSON vectors are the same as those for VARCHAR vectors, possibly via VectorReader<StringView> and StringWriter<>.